index.es.js

```diff
@@ -0,0 +1,35 @@
+import assert from 'www.unpkg.com/nanoassert?module'
```

Comment:
I think this needs a protocol?

Reply:
Something like 

```js
import assert from 'https://www.unpkg.com/nanoassert?module'
```

Reply:
yea. actually, doesn't unpkg automatically rewrite import paths when you use `?module`? just doing `import assert from 'nanoassert'` might just work already.

Reply:
It doesn't .Something like `https://unpkg.com/md?module` get resolved to `https://unpkg.com/md@0.6.0/dist/md.es.js?module` which is the actual version plus whatever you defined in your module property in package.json

Reply:
I mean how the `import` statements in the package source code get rewritten, like this:
![image](https://user-images.githubusercontent.com/1006268/47088660-5368db00-d21f-11e8-9dfb-0ee65dddc4a7.png)

so if we do `import assert from 'nanoassert'` here, unpkg would turn it into `import assert from 'https://unpkg.com/nanoassert@x.y.z/index.js?module'` automatically